tests/www/test_views.py

```diff
@@ -1534,7 +1549,7 @@ def test_dag_autocomplete_success(self):
         self.login(username='all_dag_user',
                    password='all_dag_user')
         resp = self.client.get(
-            'dagmodel/autocomplete?query=example_bash&showPaused=True',
+            'dagmodel/autocomplete?query=example_bash&status=all',
```

Comment:
This end point only looks at the cookie value (which is correct) so we shouldn't pass this argument here, but instead set a cookie on the request.

I suspect this is working right now because `home?status=all` was the last time we set set the status, and tests are currently run top-to-bottom, but if you ran just this single test on its own it might fail.

Reply:
I didn't change anything for this, but ran the single test alone and it still passed.

Reply:
Oh, the "all" is the default is why. I suspect if you added a `self.client.get('home?status=paused', follow_redirects=True)` just before this line this test would then start failing.

Reply:
Correct, this wasn't needed. Removed the param.